### PR TITLE
Feature/create new confirm account details base component

### PR DESCRIPTION
**Pull request by omarCreativeDev**

### CODE REVIEW

File: src/app/features/account/confirm-account-details/confirm-account-details.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    });
+  }
+
+  private setupSubscriptions(): void {
+    this.subscriptions.add(
+      this.userService.userDetails$.subscribe((userDetails: UserDetails) => (this.userDetails = userDetails))
+    );
+
+    this.subscriptions.add(
+      this.registrationService.selectedLocationAddress$.subscribe(
```
Comment (ChazUK) [inline src/app/features/account/confirm-account-details/confirm-account-details.ts]: Could you not do a combineLatest with all of these?

File: src/app/features/account/confirm-account-details/confirm-account-details.ts [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+          },
+        ],
+      },
+    ];
+  }
+
+  private setupServerErrorsMap(): void {
+    this.serverErrorsMap = [
+      {
+        name: 503,
```
Comment (ChazUK) [inline src/app/features/account/confirm-account-details/confirm-account-details.ts]: HTTP Status codes 500 and up are now caught by the HTTP Interceptor and redirected to the Problem With Service page

Comment (omarCreativeDev) [reply src/app/features/account/confirm-account-details/confirm-account-details.ts]: done

File: src/app/features/account/confirm-account-details/confirm-account-details.ts (hunk 1 above)
Comment (omarCreativeDev) [reply src/app/features/account/confirm-account-details/confirm-account-details.ts]: done

**Comment by omarCreativeDev** [comment]
[(13) User accounts - Create User Accounts](https://trello.com/c/rCjsbkOX/137-13-user-accounts-create-user-accounts)

**Comment by ChazUK** [comment]
[Accessibility - Inconsistent Button Markup](https://trello.com/c/EsZaRVF4/220-accessibility-inconsistent-button-markup)

**Pull request merged**